.changeset/nice-snakes-camp.md

```diff
@@ -0,0 +1,72 @@
+---
+"@salt-ds/core": minor
+---
+
+- Updated Button component CSS token values, minor visual changes expected for CTA and Primary button variants as fontweight changes from bold to semibold
```

Comment:
I'm not sure if it's worth focusing on the token impact here (core). I would instead show a diff as a picture (we can get design to provide this cc @pseys )